### ISSUE TITLE
Unable to install the package on Laravel 5.8

### ISSUE THREAD

**Issue by bistory** [lahaxearnaud/laravel-u2f#21]
Hi,

I'm trying to install this package on a up-to-date Laravel but composer (1.8.4) complains about Laravel 5.8 dependancy :

```
$ composer require lahaxearnaud/laravel-u2f
Using version ^1.4 for lahaxearnaud/laravel-u2f
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for lahaxearnaud/laravel-u2f ^1.4 -> satisfiable by lahaxearnaud/laravel-u2f[1.4.0].
    - Conclusion: remove laravel/laravel dev-master
    - lahaxearnaud/laravel-u2f 1.4.0 requires laravel/laravel ^5.8 -> satisfiable by laravel/laravel[v5.8.0, v5.8.3].
    - Can only install one of: laravel/laravel[v5.8.0, dev-master].
    - Can only install one of: laravel/laravel[v5.8.3, dev-master].
    - Installation request for laravel/laravel dev-master -> satisfiable by laravel/laravel[dev-master].


Installation failed, reverting ./composer.json to its original content.
```
My composer file is not exotic... : 

```
{
    "name": "laravel/laravel",
    "description": "The Laravel Framework.",
    "keywords": ["framework", "laravel"],
    "license": "MIT",
    "type": "project",
    "require": {
        "php": "^7.1.3",
        "doctrine/dbal": "^2.8",
        "facebook/php-ads-sdk": "^3.1",
        "fideloper/proxy": "^4.0",
        "laravel/framework": "5.8.*",
        "laravel/passport": "^7.0",
        "laravel/socialite": "^4.0",
        "laravel/tinker": "^1.0"
    },
    "require-dev": {
        "beyondcode/laravel-dump-server": "^1.0",
        "filp/whoops": "^2.0",
        "fzaninotto/faker": "^1.4",
        "mockery/mockery": "^1.0",
        "nunomaduro/collision": "^2.0",
        "phpunit/phpunit": "^7.0"
    },
    "autoload": {
        "classmap": [
            "database/seeds",
            "database/factories"
        ],
        "psr-4": {
            "App\\": "app/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "extra": {
        "laravel": {
            "dont-discover": [
            ]
        }
    },
    "scripts": {
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate"
        ],
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover"
        ]
    },
    "config": {
        "preferred-install": "dist",
        "sort-packages": true,
        "optimize-autoloader": true
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

Do you have an idea of what happens here ?
Thanks ;-)

**Comment by bistory** [comment]
Commented https://github.com/lahaxearnaud/laravel-u2f/commit/36e7a92d158d3bfb66e864a44d082198b598a821#r32875116 to fix the problem :-)

**Comment by lahaxearnaud** [comment]
Right I delete the laravel/laravel and publish a new release. Thanks for the analyse

**Comment by lahaxearnaud** [comment]
I dit not use your pull request because you didn't update the composer.lock. sorry

**Comment by bistory** [comment]
No problem, I just updated the .json quickly, forgot the .lock ;)

**Comment by lahaxearnaud** [comment]
I released the version 1.4.1

https://packagist.org/packages/lahaxearnaud/laravel-u2f

**Comment by bistory** [comment]
Thanks for the fix, it works now ;-)
